browser-test/src/applicant/questions/name.test.ts

```diff
@@ -259,6 +259,24 @@ test.describe('name applicant flow', () => {
         })
       })
 
+      test('validate name suffix field with north star flag and name suffix dropdown flag enabled', async ({
+        page,
+        applicantQuestions,
+      }) => {
+        await enableFeatureFlag(page, 'name_suffix_dropdown_enabled')
+
+        await test.step('name suffix field available to use', async () => {
+          await applicantQuestions.applyProgram(programName)
+          await expect(page.getByLabel('Suffix')).toBeVisible()
```

Comment:
Should 'Suffix' be a static constant?

Reply:
I don't think we're going to change the label anytime soon, plus this is in test. But good point taken!